lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -64,9 +65,10 @@ function requestHandler(request, response) {
     }
     response.writeHead(statusCode, headers);
 
-    if (queryString && queryString.includes('delay')) {
+    if (queryString && typeof queryString.delay !== 'undefined') {
```

Comment:
maybe add a comment here for future folks about how this will work? You can set the value for `delay` or any non-numeric value will default to 2000 ms?